docs/sources/panels/visualizations/node-graph.md

```diff
@@ -45,17 +45,37 @@ You can pan and zoom in or out the node graph.
 
 ### Pan
 
-You can pan the view by clicking outside of any node or edge and dragging your mouse.
+You can pan the view by clicking outside any node or edge and dragging your mouse.
 
 ### Zoom in or out
 
-Use the buttons on the upper left corner or use the mouse wheel, touch pad scroll, together with either Ctrl or Cmd key to zoom in or out.
+Use the buttons in the upper left corner or use the mouse wheel, touchpad scroll, together with either Ctrl or Cmd key to zoom in or out.
+
+### Explore hidden nodes
+
+Number of nodes shown at a given time is limited to maintain reasonable performance of the visualisation. Nodes that are not currently visible are hidden behind clickable markers that show approximate number of hidden nodes that are connected by particular edge. You can click on the marker to expand the graph around that node.
+
+![Node graph exploration](/static/img/docs/node-graph/node-graph-exploration-8-0.png "Node graph exploration")
+
+### Grid view
+
+You can switch to grid view to have a better overview of the most interesting nodes in the graph. Grid view show nodes in a grid without edges and can be sorted by stats shown inside the node or by stats represented by colored border of the nodes.
+
+![Node graph grid](/static/img/docs/node-graph/node-graph-grid-8-0.png "Node graph grid")
+
+To sort the nodes click the on the stats inside the legend. Marker next to the stat name shows which stat is currently used for sorting and sorting direction.
+
+![Node graph legend](/static/img/docs/node-graph/node-graph-legend-8-0.png "Node graph legend")
+
+Click on the node and select "Show in Graph layout" option to switch back to default layout and focus on the selected node, to show it in context of the full graph.
```

Comment:
Not clear what is being done. Maybe define "default" layout and mention this step will help you switch back to default layout.

Reply:
Maybe `to switch back to graph layout`? We already have graph layout in the button/link so maybe that would be better. I use "default" or "graph" layout for the initial way the graph is shown as opposite to "grid" layout to which you can explicitly switch.

Reply:
Yes, makes sense. Thanks!